src/Blocks/custom/featured-posts/featured-posts.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/**
+ * Template for the Featured Posts view.
+ *
+ * @package Unicorns
+ */
+
+use UnicornsVendor\EightshiftLibs\Helpers\Components;
+
+$manifest = Components::getManifest(__DIR__);
+
+$blockClass = Components::checkAttr('blockClass', $attributes, $manifest);
+$query = Components::checkAttr('query', $attributes, $manifest);
+$excludeCurrentPost = Components::checkAttr('excludeCurrentPost', $attributes, $manifest);
+$itemsPerLine = Components::checkAttr('itemsPerLine', $attributes, $manifest);
+$showItems = Components::checkAttr('showItems', $attributes, $manifest);
+$serverSideRender = Components::checkAttr('serverSideRender', $attributes, $manifest);
+
+global $post;
+
+?>
+
+<div class="<?php echo esc_attr($blockClass); ?>" data-items-per-line=<?php echo \esc_attr($itemsPerLine); ?>>
+    <?php
+$request  = new \WP_REST_Request('GET', '/eightshift-libs/v1/news');
+$request->set_param('_limit', 4);
```

Comment:
not a big fan of this implementation. You are throttling the whole site until Rest request is resolved. 

I think this implementation should stay the same as it was from the libs and just provide the Load more component necessary operators to make a query on load more using JS